apps/server/src/modules/board/uc/board.uc.ts

```diff
@@ -91,13 +92,14 @@ export class BoardUc {
 		await this.columnService.updateTitle(column, title);
 	}
 
-	async createCard(userId: EntityId, columnId: EntityId): Promise<Card> {
+	async createCard(userId: EntityId, columnId: EntityId, createCardBodyParams?: CreateCardBodyParams): Promise<Card> {
```

Comment:
also here, we can use requiredEmptyElements like in the service
this way, we don't care about validation, or the api call itself